src/server/connection.rs

```diff
@@ -658,6 +661,14 @@ impl Connection {
                                     conn.on_close("stop service", false).await;
                                     break;
                                 }
+                                Some(misc::Union::WindowFocus(_)) => {
+                                    #[cfg(not(any(target_os = "android", target_os = "ios")))]
+                                    {
+                                        if conn.follow_remote_window {
+                                            conn.handle_window_focus().await;
+                                        }
+                                    }
+                                }
```

Comment:
Multiple problems here:
- [ ] since WindowFocus value is not used, why do you use uint?
- [ ] should `continue` here because this message is not needed to passed to peer
- [ ] I do not like your implementation here. Since it is a service, you should unsubscribe it if `get_subbed_displays_count` > 1 rather than creating additional protobuf message